sys/linux/socket_netlink_route.txt

```diff
@@ -758,6 +799,53 @@ ifa_cacheinfo {
 	tstamp		int32
 }
 
+br_vlan_db_dump_pol [
+	BRIDGE_VLANDB_DUMP_FLAGS	nlattr[BRIDGE_VLANDB_DUMP_FLAGS, flags[br_vlan_dumpflags, int32]]
+]
+
+br_vlan_db_policy [
+	BRIDGE_VLANDB_ENTRY	nlnest[BRIDGE_VLANDB_ENTRY, br_vlan_db_policy_entry]
+] [varlen]
+
+br_vlan_db_policy_entry [
```

Comment:
Please name this one "br_vlan_db_policy", that's how it's called in the kernel:
https://elixir.bootlin.com/linux/v5.8-rc4/source/net/bridge/br_vlan.c#L1886
or syz-check is confused. And rename br_vlan_db_policy to something else.

Reply:
Done, let me know what you think.